data/json/recipes/recipe_vehicle.json

```
@@ -452,6 +452,22 @@
     "qualities": [ { "id": "DRILL", "level": 3 } ],
     "components": [ [ [ "scrap", 2 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "turret_mount",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_VEHICLE",
+    "skill_used": "mechanics",
+    "difficulty": 2,
+    "time": "30 m",
+    "autolearn": true,
+    "using": [ [ "welding_standard", 3 ] ],
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "SAW_M", "level": 1 }, { "id": "WRENCH", "level": 1 } ],
+    "components": [
+      [ [ "pipe", 3 ] ],
+      [ [ "steel_chunk", 4 ], [ "scrap", 12 ] ],
```

```suggestion
      [ [ "steel_chunk", 4 ], [ "scrap", 12 ] ]
```
No `,` at the end of an array